monai/utils/type_conversion.py

```diff
@@ -214,4 +214,12 @@ def convert_to_dst_type(src: Any, dst: NdarrayOrTensor) -> Tuple[NdarrayOrTensor
     device = None
     if isinstance(dst, torch.Tensor):
         device = dst.device
-    return convert_data_type(data=src, output_type=type(dst), device=device, dtype=dst.dtype)
+
+    output_type: Any
```

Comment:
could you please help update the docstring and the test `tests/test_convert_data_type.py`

Reply:
updated in the latest commit.
Thanks.